### ISSUE TITLE
RtContainer.java:42-44: Keep implementing the rest of the...

### ISSUE THREAD

**Issue by 0pdd** [amihaiemil/docker-java-api#76]
The puzzle `57-4a6d0f49` from #57 has to be resolved:

https://github.com/amihaiemil/docker-java-api/blob/1cf306497763afee903a62fc853df37b15ff9c88/src/main/java/com/amihaiemil/docker/RtContainer.java#L42-L44

The puzzle was created by George Aristy on 03-Apr-18. 

Estimate: 30 minutes, role: DEV.

If you have any technical questions, don't ask me, submit new tickets instead. The task will be "done" when the problem is fixed and the text of the puzzle is _removed_ from the source code. Here is more about [PDD](http://www.yegor256.com/2009/03/04/pdd.html) and [about me](http://www.yegor256.com/2017/04/05/pdd-in-action.html).

**Comment by 0crat** [comment]
@amihaiemil[/z](https://www.0crat.com/u/amihaiemil) please, pay attention to this issue

**Comment by 0crat** [comment]
@0pdd[/z](https://www.0crat.com/u/0pdd) this project will fix the problem faster if you donate a few dollars to it; just [click here](https://www.0crat.com/contrib/G6LPQQV2P) and pay via Stripe, it's very fast, convenient and appreciated; thanks a lot!

**Comment by 0crat** [comment]
Job [#76](https://github.com/amihaiemil/docker-java-api/issues/76) is now in scope, role is `DEV`

**Comment by 0crat** [comment]
The job [#76](https://github.com/amihaiemil/docker-java-api/issues/76) assigned to @llorllale[/z](https://www.0crat.com/u/llorllale), here is [why](https://www.0crat.com/footprint/G6LPQQV2P/941444710); the budget is 30 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§8](http://www.zerocracy.com/policy.html#8) and [§9](http://www.zerocracy.com/policy.html#9); if the task is not clear, read [this](http://www.yegor256.com/2015/02/16/it-is-not-a-school.html) and [this](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html)

**Comment by 0crat** [comment]
@llorllale[/z](https://www.0crat.com/u/llorllale) this job was assigned to you 5days ago. It will be taken away from you soon, unless you close it, see [§8](http://www.zerocracy.com/policy.html#8). Read [this](http://www.yegor256.com/2014/04/13/no-obligations-principle.html) and [this](http://www.yegor256.com/2014/11/24/principles-of-bug-tracking.html), please.

**Comment by llorllale** [comment]
@0crat waiting on merge for #80

**Comment by 0crat** [comment]
> @0crat waiting on merge for #80  ([here](https://github.com/amihaiemil/docker-java-api/issues/76#issuecomment-381334375))

@llorllale The impediment for [#76](https://github.com/amihaiemil/docker-java-api/issues/76) was registered successfully by @llorllale[/z](https://www.0crat.com/u/llorllale)

**Comment by llorllale** [comment]
@amihaiemil something wrong with @0pdd? This should've been closed, I believe, when #80 was merged.

**Comment by amihaiemil** [comment]
@llorllale I think there is that puzzle you said was malformed a few days ago.
I received this via email: 

```
ERROR: src/test/java/com/amihaiemil/docker/mock/Response.java; puzzle at line #54; @todo found, but puzzle can't be parsed, most probably because @todo is not followed by a puzzle marker, as this page explains: https://github.com/yegor256/pdd#how-to-format. If you can't understand the cause of this issue or you don't know how to fix it, please submit a GitHub issue, we will try to help you: https://github.com/yegor256/pdd/issues. This tool is still in its beta version and we will appreciate your feedback. Here is where you can find more documentation: https://github.com/yegor256/pdd/blob/master/README.md.
Exit code is 1
```

**Comment by amihaiemil** [comment]
@llorllale I'll fix that puzzle later, apparently I cannot edit the file on-the-fly. It is indeed missing the ``#``.

**Comment by 0crat** [comment]
Order was finished: +30 point(s) just awarded to @llorllale[/z](https://www.0crat.com/u/llorllale)

**Comment by 0crat** [comment]
The job [#76](https://github.com/amihaiemil/docker-java-api/issues/76) is now out of scope

**Comment by 0pdd** [comment]
@0pdd the puzzle [#88](https://github.com/amihaiemil/docker-java-api/issues/88) is still not solved.

**Comment by 0pdd** [comment]
@0pdd the puzzle [#97](https://github.com/amihaiemil/docker-java-api/issues/97) is still not solved; solved: [#88](https://github.com/amihaiemil/docker-java-api/issues/88).
